Make links more visibles

CharlyCst opened the pull request:
The link color was black, due to the template's CSS reset, which made
them hard to distinguish from normal text. This commit makes the links
green (the theme's color) to make them easier to distinguish

Before:

<img width="617" alt="Screenshot 2023-06-10 at 18 51 11" src="https://github.com/jsysresearch/website/assets/23420125/8fd863a6-a9e1-45f5-9e38-846eb86705d0">

After:
<img width="596" alt="Screenshot 2023-06-10 at 19 24 49" src="https://github.com/jsysresearch/website/assets/23420125/a4e84eef-16d5-48f5-806f-6e4a92cfbfb1">

(This PR can be rebased on top of #31)

romain-jacob commented:
That's fine with me. It used to be like that in the past, actually, but @vijay03 didn't like it too much, if I remember well (though I'm not too sure TBH). 

@vijay03 ?

CharlyCst commented:
I see, another solution would be to underline links, like I do on [my website](https://charlycst.github.io/) for instance, would that work better?

I feel like the current version is not great accessibility wise. I barely can distinguish links from normal text myself and I often discover them by hovering over with the mouse. Addressing this would definitely make the website better in my opinion.

vijay03 commented:
I would agree with underlining the links too


On Sun, Sep 17, 2023, 1:30 PM Charly ***@***.***> wrote:

> I see, another solution would be to underline the links, like I do on my
> website <https://charlycst.github.io/> for instance, would that work
> better?
>
> I feel like the current version is not great accessibility wise. I barely
> can distinguish links from normal text myself and I often discover them by
> hovering over with the mouse. Addressing this would definitely make the
> website better in my opinion.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jsysresearch/website/pull/32#issuecomment-1722561655>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAHKPJRFG44BBFGDMYHLEFDX25MUPANCNFSM6AAAAAAZBZPM7Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

romain-jacob commented:
Fine with me.

CharlyCst committed:
Accessibility: make links more visible

The links in the page content are now underlined. In addition, they
change color (to green primary color) on hover to indicate that they are
clickable.

CharlyCst commented:
Updated the PR to underline links, instead of changing color to green. This only affect links in the page content, not in menus.

![image](https://github.com/jsysresearch/website/assets/23420125/e726233a-c403-4bea-9478-a3ed78b84866)

I also uncommented to hover attribute of the template, this makes links green *when hovering with the mouse* to make it even clearer that links are clickable. This doesn't change anything on mobile and doesn't change the overall aspect of the page, I hope it's fine.

romain-jacob commented:
It's great, thanks!
